modules/virt-define-guest-agent-ping-probe.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+:_content-type: PROCEDURE
+[id="virt-define-guest-agent-ping-probe_{context}"]
+
+= Defining a guest agent ping probe
+
+Define a guest agent ping probe by setting the `spec.readinessProbe.guestAgentPing` field of the virtual machine instance (VMI) configuration.
+
+:FeatureName: The guest agent ping probe
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* The QEMU guest agent must be installed and enabled on the virtual machine.
+
+.Procedure
+
+. Include details of the guest agent ping probe in the VMI configuration file.
++
+
+.Sample guest agent ping probe
+[source,yaml]
+----
+# ...
+spec:
+  readinessProbe:
+    guestAgentPing: {} <1>
+    initialDelaySeconds: 120 <2>
+    periodSeconds: 20 <3>
+    timeoutSeconds: 10 <4>
+    failureThreshold: 3 <5>
+    successThreshold: 3 <6>
```

Comment:
Maybe a complete yaml would be better.

Reply:
that means $ oc create -f <file_name>.yaml --> mentioned in line no 48 is not valid here. 
Should we explicitly mentioned what is supported and the above ticket is in progress

Reply:
It is already inconsistent in the docs currently. The correct flow would be to do this in the UI, which is more hard to describe. I would just leave it as it is.